lms/djangoapps/discussion_api/api.py

```diff
@@ -255,9 +255,12 @@ def get_thread_list(
         view=None,
         order_by="last_activity_at",
         order_direction="desc",
+        discussion_id=None,
```

Comment:
If it doesn't cause a lot of breakage, consider moving the `discussion_id` parameter right below the `topic_id_list` parameter since they are both filters.